db/schema.rb

```diff
@@ -11,7 +11,7 @@
 # It's strongly recommended that you check this file into your version control system.
 
 ActiveRecord::Schema[7.2].define(version: 2024_10_10_165823) do
-  create_table "active_storage_attachments", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
+  create_table "active_storage_attachments", charset: "utf8mb4", collation: "utf8mb4_general_ci", force: :cascade do |t|
```

Comment:
These changes to `schema.rb` are unrelated and something odd about your development setup. Could you paste the output of `mariadb --version`? We see this occasionally and I'd like to understand what's different so I can gave better guidance to future contributors or improve our setup instructions in the README.

Reply:
Actually, I don't know why this has been changed. Here is the output of mariadb --version : mariadb  Ver 15.1 Distrib 10.3.39-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2 
I don't know if it's related but i'm using a WSL 2 virtual linux environment to run the project.

Reply:
That's really odd. I spent a few minutes searching and the difference between the two is an (outdated?) minor performance improvement. I'd like to keep the value we have in production in this file. Sorry for the hassle, but please do revert these collation changes.

Reply:
@chamlis figured it out:

> it seems upstream MariaDB defaults to latin1_swedish_ci, but Debian/Ubuntu patch it to default to utf8mb4_general_ci and Arch patches it to default to utf8mb4_unicode_ci

So _I'm_ the one introducing the incorrect value when I'm editing on my local dev, which runs Arch. I'll fix it on `master`.